src/orion/algo/space.py

```diff
@@ -475,7 +461,7 @@ def interval(self, alpha=1.0):
             int_high = int(numpy.floor(high))
         except OverflowError:  # infinity cannot be converted to Python int type
             int_high = numpy.inf
-        if int_high < high:  # Exclusive upper bound
+        if int_high < high:  # Inclusive upper bound
```

Comment:
Why is it not `<=` ?

Reply:
Because we would overflow to next integer when `int_high == high`. That should be removed actually and use `ceil` instead of `floor` for `int_high`.

Reply:
Thanks for the explanations!